experimental/regression_suite/shark-test-suite-models/sd3/test_clip.py

```diff
@@ -0,0 +1,167 @@
+    group="sd3_clip",
+)
+
+sd3_clip_inference_output_1 = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/inference_output.1.bin",
+    group="sd3_clip",
+)
+
+sd3_clip_real_weights = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/real_weights.irpa",
+    group="sd3_clip",
+)
+
+sd3_clip_mlir = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/model.mlirbc",
+    group="sd3_clip",
+)
+
+CPU_COMPILE_FLAGS = [
+    "--iree-hal-target-backends=llvm-cpu",
+    "--iree-llvmcpu-target-cpu-features=host",
+    "--iree-llvmcpu-fail-on-out-of-bounds-stack-allocation=false",
+    "--iree-llvmcpu-distribution-size=32",
+    "--iree-opt-const-eval=false",
+    "--iree-llvmcpu-enable-ukernels=all",
+    "--iree-global-opt-enable-quantized-matmul-reassociation",
+]
+
+
+@pytest.fixture
+def SD3_CLIP_COMMON_RUN_FLAGS(
+    sd3_clip_inference_input_0,
+    sd3_clip_inference_input_1,
+    sd3_clip_inference_input_2,
+    sd3_clip_inference_input_3,
+    sd3_clip_inference_input_4,
+    sd3_clip_inference_input_5,
+    sd3_clip_inference_output_0,
+    sd3_clip_inference_output_1,
+):
+    return [
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_0.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_1.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_2.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_3.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_4.path}",
+        f"--input=1x77x2xi64=@{sd3_clip_inference_input_5.path}",
+        f"--expected_output=2x154x4096xf32=@{sd3_clip_inference_output_0.path}",
+        f"--expected_output=2x2048xf32=@{sd3_clip_inference_output_1.path}",
+    ]
+
+
+ROCM_COMPILE_FLAGS = [
+    "--iree-hal-target-backends=rocm",
+    f"--iree-rocm-target-chip={rocm_chip}",
+    "--iree-input-type=torch",
+    "--iree-opt-const-eval=false",
+    "--iree-global-opt-propagate-transposes=true",
+    "--iree-opt-outer-dim-concat=true",
+    "--iree-rocm-waves-per-eu=2",
+    "--iree-llvmgpu-enable-prefetch",
+    "--iree-flow-enable-aggressive-fusion",
+    "--iree-global-opt-enable-fuse-horizontal-contractions=true",
+    "--iree-opt-aggressively-propagate-transposes=true",
+    "--iree-codegen-llvmgpu-use-vector-distribution=true",
+    "--iree-execution-model=async-external",
+    "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, util.func(iree-preprocessing-pad-to-intrinsics{pad-target-type=conv}))",
+]
+
+###############################################################################
+# CPU
+###############################################################################
+
+
+def test_compile_clip_cpu(sd3_clip_mlir):
+    VmfbManager.sd3_clip_cpu_vmfb = iree_compile(
+        sd3_clip_mlir, "cpu", CPU_COMPILE_FLAGS
+    )
```

Comment:
How does this differ from the fixture approach taken here?
https://github.com/iree-org/iree/blob/6df0372e67b022f021668f3da1c8d9f13495d2f6/experimental/regression_suite/tests/pregenerated/test_llama2.py#L97-L107 https://github.com/iree-org/iree/blob/6df0372e67b022f021668f3da1c8d9f13495d2f6/experimental/regression_suite/tests/pregenerated/test_llama2.py#L220-L240

Reply:
In the previous approach, we only had the running part as the tests. Now, we make the compilation part tests also. They wouldn't show as tests in the log if we make compilation a fixture, which I think is pretty valuable for anyone running these tests.